CONTRIBUTING.md

```diff
@@ -11,6 +11,11 @@ Instead, modify the sources in the ```www``` folder and compile a new version of
 
 ```node.js``` and ```npm``` are needed for building this project. ```npm``` comes bundled with the ```node.js``` installer. You can download the ```node.js``` installer here: https://nodejs.org/download/. 
 
+To run Android tests, make sure you have ```sdk\tools``` and  ```sdk\platform-tools``` in your PATH.
+
+If you would like the tests to start the emulators for you, configure an Android emulator using the AVD and name it ```emulator```.
```

Comment:
i will keep the task names as is just because i think it works better for actual execution (ie "test" is shorthand for "test --android --ios-ui --ios-wk") but emulator names passed by command line sounds groovy